lib/controllers/users.js

```diff
@@ -4,21 +4,153 @@ var express         = require("express"),
     errors          = require("../errors.js").ERRORS,
     request         = require("request"),
     config          = require("../../config"),
+    uuidv4          = require("uuid/v4"),
     authenticate    = require("./helpers/auth.js").authenticate;
 
 var users = module.exports = express.Router({ mergeParams: true });
 
 var User = mongoose.model("User");
 var Patient = mongoose.model("Patient");
 
+
+const makeRequest = (data, callback) => {
+  request.post(data, callback);
+};
+const getRequest = (data, callback) => {
+  request.get(data, callback);
+};
+const deleteRequest = (data, callback) => {
+  request.delete(data, callback);
+};
+
+// Add UUID for users that do not already have one on their account
+User.find({ "uuid" : { "$exists" : false } }, function(err, users) {
+  if (err) {
+     console.log(err);
+  } else if (users.length > 0) {
+        var adminToken = null;
+        var adminId = null;
+        var usersUpdated = 0;
+        const adminUser = {
```

Comment:
NBD but we may not need to create this user since we already have the microservice user with admin permissions